wiki/BBCode/de.md

```diff
@@ -0,0 +1,304 @@
+
+Das zuvor genannte Argument darf keine Anführungszeichen (`"`) enthalten und hat keinen Standardwert. Wenn kein Wert festgelegt wurde oder, wenn Anführungszeichen benutzt werden, dann wird das Tag nicht als BBCode-Tag verarbeitet.
+
+### Schriftgröße
+
+```
+[size=NUMBER]Text[/size]
+```
+
+Das Tag `[size]` wird benutzt, um Text durch verschiedene Schriftgrößen zu gestalten. Momentan werden vier Größen von der osu!-Webseite unterstützt: 50, 85, 100, und 150. Die Größen werden intern als "winzig", "klein", "normal" und "groß" entsprechend bezeichnet.
+
+Das Argument `NUMBER` akzeptiert keine Anführungszeichen und nimmt nur eine der vier unterstützten Größen an. Wenn eine Nummer eingeben wird, die nicht eine der vier unterstützten entspricht, dann wird die Textgröße auf den Standardwert zurückgesetzt.
+
+Schaltfläche in der Symbolleiste: ![Optionen bei der Schriftgröße](img/font-size-DE.png "Schriftgröße")
+
+### Spoiler
+
+*Nicht zu verwechseln mit [Spoilerbox](#spoilerbox).*
+
+```
+[spoiler]Text[/spoiler]
+```
+
+Das Tag `[spoiler]` wird benutzt, um sensitive Daten mit einem schwarzen Vordergrund zu verdecken. Diese Daten werden beim Markieren sichtbar. Wenn der Vordergrund mit dem Farb-Tag [`[color]`](#farbe) gestapelt wird, dann wird der schwarze Vordergrund davon nicht beeinflusst. Allerdings wird der Text unter dem schwarzen Vordergrund unabhängig von der Lesbarkeit gefärbt sein.
+
+Das Tag wird meistens benutzt, um kritische/sensitive Informationen über eine TV-Show, eines Films oder anderer Medien zu verstecken, damit Leute nicht gespoilert werden. Es wird manchmal auch verwendet, um einen komödiantischen Effekt oder eine Betonung zu erzeugen.
+
+### Box
+
+*Nicht zu verwechseln mit [Spoilerbox](#spoilerbox).*
+
+```
+[box=NAME]
+Text
+[/box]
+```
+
+Das Tag `[box]` wird verwendet, um Texte und Bilder in einem anklickbaren Hyperlink zu verstecken. Beim Klicken wird der Inhalt in der Box ähnlich wie in einem Dropdown-Menü aufgedeckt.
+
+Der benutzerdefinierte Hyperlink wird durch das Argument `NAME` bestimmt. Die Spezifizierung dieses Arguments wird eine Überschrift in der Box anhand des Arguments erstellen und wird die Größe der Box entsprechend anpassen. Wenn die Spezifizierung nicht erfolgt, dann wird das Tag `[box]` standardmäßig eine Box ohne Überschrift erstellen (auf die nicht geklickt werden kann). Das Argument benutzt keine Anführungszeichen (`"`) und stellt Leerzeichen dar.
+
+Das Tag wird meistens benutzt, um große Fluten an Texten und Bildern zu verstecken, die den Forum-Beitrag immens verlängern. Am meisten sichtbar in den FAQ oder in [Skin](/wiki/Skinning)-Veröffentlichungsbeiträgen.
```

Comment:
```suggestion
Das Tag wird meistens benutzt, um große Fluten an Text und Bildern zu verstecken, die den Forum-Beitrag immens verlängern. Am meisten sichtbar in den FAQ oder in [Skin](/wiki/Skinning)-Veröffentlichungsbeiträgen.
```

In the context of formatting a single post/description/etc. you would mostly be talking about text in singular form(referring to the entire post as "the text").